.github/actions/run-e2e-playwright/action.yml

```diff
@@ -28,3 +28,11 @@ runs:
         echo "Running playwright tests with options ${{ inputs.options }}"
         sleep 90
         npx nx run ${{ inputs.options }}-e2e:e2e
+
+    - name: Upload E2Es results
```

Comment:
Overall it looks okay to me but please keep an eye on this action, in the past there were some strange issues when this action got frozen and was running for couple of hours

Reply:
Yes, I will keep an eye on this and will change settings if needed. For now lets leave it as is